app/models/record.server.ts

```diff
@@ -7,6 +7,13 @@ import type { Record } from '@prisma/client';
 export const createUserRecord = async (
   data: Pick<Record, 'username' | 'type' | 'subdomain' | 'value'>
 ) => {
+  if (!process.env.MAX_USER_RECORD_LIMIT) {
```

Comment:
Let's not hard-code a default limit here.  If the env is set, then we do 13-15, otherwise we skip it (i.e., we only enforce a limit if requested).